### ISSUE TITLE
Sensor docker.port.XX not being mapped 

### ISSUE THREAD

**Issue by johnmccabe** [brooklyncentral/clocker#280]
Adding an app using the following YAML successfully deploys but the `docker.port.3306` sensor does have a corresponding `mapped.docker.port.3306` added (its absent rather than empty), looks like the `hostAndPortTransformingEnricher` isn't being added.
```yaml
location: dc
services:
- type: docker:mariadb:5.5
  id: db
  containerName: mysql
  env:
    MYSQL_ROOT_PASSWORD: "password"
``` 
The container **does** get created correctly, and the mapped port is there and accessible just not published in a sensor.

**Comment by grkvlt** [comment]
Trying some changes in ab0a29e5084ba2603c66cd4ca5b92c9102940647 that may fix this

**Comment by johnmccabe** [comment]
Thanks @grkvlt this looks good now.

**Comment by johnmccabe** [comment]
Closing as fixed in #272
